<launch>
    <param name="robot_name" value="zoidstein" />
    <param name="armature_name" value="Armature" />
    <param name="joint_axes_yaml" value="$(find ros_blender_bridge)/launch/zoidstein_joint_axes.yaml" />

    <node pkg="ros_blender_bridge" type="launch_blender" name="launch_blender" output="screen">
        <param name="blend_file" value="$(find ros_blender_bridge)/launch/zoidstein.blend" />
        <param name="use_game_engine" value="false" />
        <param name="python_script" value="$(find ros_blender_bridge)/scripts/run_bpy_joint_trajectory_server.py" />
    </node>
</launch>